eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -334,14 +354,17 @@ govTransition = do
               == expectedDep
                 ?! ProposalDepositIncorrect pProcDeposit expectedDep
 
+        -- Policy check
+        runTest $ checkPolicy constitutionPolicy pProcGovAction
```

Comment:
There is already pattern matching on GovActions happens few lines below. Maybe turn `checkPolicy` into?
```haskell
checkPolicy :: StrictMaybe (ScriptHash (EraCrypto era)) -> StrictMaybe (ScriptHash (EraCrypto era)) -> Test (ConwayGovPredFailure era)
```

Just a suggestion, doesn't make too much difference if you decide to keep it the way it is now.